src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -524,13 +524,18 @@ public function makePathRelative($endPath, $startPath)
     /**
      * Mirrors a directory to another.
      *
+     * All files and directories in the origin directory will be copied into the target directory. By default:
+     *
+     * - existing files in the target directory will be overwritten, except if they are newer (see the `override` option)
```

Comment:
the bullet misses a space before (one for the indentation of the docblock, one for the indentation of the bullet)

Reply:
Fixed in #26215